### PR TITLE
Publish updates from master

### PR DESCRIPTION
**Pull request by usha-mandya**
Sync the latest changes from master

### CODE REVIEW

File: engine/install/ubuntu.md [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
 redirect_from:
 - /engine/installation/ubuntulinux/
 - /installation/ubuntulinux/
+- /engine/installation/linux/ubuntu/
 - /engine/installation/linux/ubuntulinux/
 - /engine/installation/linux/docker-ce/ubuntu/
 - /install/linux/ubuntu/
-- /engine/installation/linux/ubuntu/
-title: Get Docker Engine - Community for Ubuntu
+- /install/linux/docker-ce/debian/
```
Comment (StefanScherer) [inline engine/install/ubuntu.md:11]: I tried https://docs-stage.docker.com/install/linux/docker-ce/ubuntu/ but it gives me a 404.

Should it be?
```suggestion
- /install/linux/docker-ce/ubuntu/
```

Comment (thaJeztah) [reply engine/install/ubuntu.md:11]: arf, yes, this one should be ubuntu; let me open a quick PR

Comment (StefanScherer) [reply engine/install/ubuntu.md:11]: Maybe there are similar redirects, it was just the first one I tried.

Comment (thaJeztah) [reply engine/install/ubuntu.md:11]: @StefanScherer opened https://github.com/docker/docker.github.io/pull/10583

Comment (usha-mandya) [reply engine/install/ubuntu.md]: > Maybe there are similar redirects, it was just the first one I tried.

Thanks Stefan. Verified the redirects from `/install/linux/` and they all point to the right topics now under `/engine/`.

**Pull request merged**